internal/gcsx/random_reader.go

```diff
@@ -211,19 +214,18 @@ func (rr *randomReader) tryReadingFromFileCache(ctx context.Context,
 
 		cacheHit = false
 		n = 0
+
 		if util.IsCacheHandleInvalid(err) {
-			err = rr.fileCacheHandle.Close()
-			if err != nil {
-				err = fmt.Errorf("tryReadingFromFileCache: while closing the fileCacheHandle: %v", err)
-				return
-			}
+			logger.Tracef("Closing cacheHandle:%p for object: %s//:%s", rr.fileCacheHandle, rr.bucket.Name(), rr.object.Name)
```

Comment:
should we include this inside CacheHandle.Close ?

Reply:
Yeah, that's the ideal place. But we don't have bucketName and objectName.